tests/t184_arg_enum.py

```diff
@@ -29,6 +29,7 @@ def sort(self, output):
                 continue
             line = ln.split('|', 1)[-1]
             func = re.sub(r'0x[0-9a-f]+', '0xADDR', line)
+            func = re.sub(r'&[_a-z]+\+[0-9]+', '0xADDR', func)
```

Comment:
It like papering over the problem.  We should fix it not to print 'variable + offset' for those address.